recipe/meta.yaml

```diff
@@ -10,32 +10,39 @@ source:
   sha256: b8b450ea8e8e3c239e9e7e38d12fed934e77a353c14b297f8ee345a5ceb25b91
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   number: 0
 
 requirements:
   host:
-    - python >=3.5,<4.0
+    - python
     - setuptools
     - wheel
     - pip
   run:
-    - python >=3.5,<4.0
+    - python
     - json-stream-rs-tokenizer >=0.4.17
 
 test:
   imports:
     - json_stream
-  commands:
-    - pip check
   requires:
     - pip
+  commands:
+    - pip check
```

Comment:
WOUld it be hard to run the tests (https://github.com/daggaz/json-stream/tree/master/src/json_stream/tests)?